### ISSUE TITLE
PiAware - fa-mlat-client "permission denied"

### ISSUE THREAD

**Issue by ShoGinn** [TheBiggerGuy/docker-ads-b#6]
I came upon your code, (thanks btw for giving me new ways to use travis ;) ).

But thats not the point, I built the PiAware image and the mlat client keeps getting permission denied.

The one thing I changed was to version 3.6.3 of PiAware, so not sure what has changed since 3.5.3.

The ADSB-Exchange one is fine. I have been able to diagnose down to the Debian image and Python versioning as the issue.

I ended up just using there Debian builder but really like the freedom of your build to control the code.

**Comment by SingingDwarf** [comment]
I am getting the exact same thing with the original 3.5.3 version of PiAware and am unable to diagnose the reason:

```
Waiting for dump1090 to start up,
piaware version 3.5.3 is running, process ID 10,
your system info is: Linux 5def6f9f97f9 4.14.98-v7+ #1200 SMP Tue Feb 12 20:27:48 GMT 2019 armv7l GNU/Linux,
Connecting to FlightAware adept server at piaware.flightaware.com/1200,
Connection with adept server at piaware.flightaware.com/1200 established,
TLS handshake with adept server at piaware.flightaware.com/1200 completed,
FlightAware server certificate validated,
encrypted session established with FlightAware,
adept reported location: 52.xxxxx, 1.xxxxx, xxxft AMSL,
my feeder ID is xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx,
site statistics URL: https://flightaware.com/adsb/stats/user/XXXXXXX#stats-xxxxxx,
logged in to FlightAware as user xxxxxx,
multilateration data requested,
Starting multilateration client: /usr/lib/piaware/helpers/fa-mlat-client --input-connect dump1090:30005 --input-type auto --results beast,listen,30105 --udp-transport 70.42.6.254:8247:656728138,
mlat-client(18): /usr/lib/piaware_packages/helpers/droproot: exec of /usr/lib/piaware/helpers/fa-mlat-client failed: Permission denied,
fa-mlat-client exited with EXIT 47,
got EOF from multilateration client
```

**Comment by TheBiggerGuy** [comment]
Thanks for reporting this issue and sorry it has taken me this long to get back to you.
I am just starting to investigate this.

**Comment by TheBiggerGuy** [comment]
This should now be fixed on the Dev branch in source however I am now fixing the Travis build to get the Dev branch to push an image to DockerHub.

**Comment by TheBiggerGuy** [comment]
Unfortunately the issue still appears and is definitely the same as #12. I will try and fix #12 first as it is a lot simpler and should be faster to root cause. Also getting Travis working again for FlightAware is a priority.
